multi-node/aws/cmd/kube-aws/command_up.go

```diff
@@ -14,8 +14,12 @@ import (
 
 	"github.com/spf13/cobra"
 
+	"archive/tar"
+	"compress/gzip"
+	"encoding/base64"
 	"github.com/coreos/coreos-kubernetes/multi-node/aws/pkg/cluster"
 	"github.com/coreos/coreos-kubernetes/multi-node/aws/pkg/tlsutil"
+	"io"
```

Comment:
Import ordering is off. Can you group stdlib first, then package specific, then other. See: https://github.com/golang/go/wiki/CodeReviewComments#imports